### DISCUSSION TITLE
BFA logo for GitHub

### DISCUSSION THREAD

**Discussion by ghost** [Bforartists/Bforartists#2567] [General]
Had this idea for a while, it would be nice if the github had Bforartists logo :)

**Comment by ReinerBforartists** [comment]
Is this even possible? I need to have a look how far the github page can be customized. Thanks for the idea :)

**Comment by ghost** [comment]
I think so yeah, I meant the profile picture, a banner image would be great too, I think it is in repo settings

**Comment by ghost** [comment]
![image](https://user-images.githubusercontent.com/25552173/127482532-f570a28f-99cc-4b8c-874d-f5ec251decfe.jpeg)

**Comment by ReinerBforartists** [comment]
Should be fixed.

I also had a look for further customizations. But that's a can of worms, and i have enough burdens already :)

**Comment by ghost** [comment]
Fixed! :) ![image](https://user-images.githubusercontent.com/25552173/127614803-e91d300f-0cf7-4a1e-89d2-e0ad57e1b632.jpeg)
